README.md

```diff
@@ -10,6 +10,24 @@ The CLASS web application is available at https://classmodel.github.io/class-web
 
 For more information on CLASS, see https://classmodel.github.io/.
 
+## Command line usage
+
+The class model can be run from the command line.
+The argument is the config file that should adhere to the [JSON schema](./packages/class/src/config.json).
+
+```shell
+pnpx --package=@classmodel/class class config.json
```

Comment:
dunno, I can do publish with a beta version to find out

Reply:
Seems not yet, waiting for `https://github.com/jsr-io/jsr/pull/284` to be merged.

Reply:
But deno can execute bin from npmjs, see https://docs.deno.com/runtime/fundamentals/node/#executable-npm-scripts. 

So we need to move cli to its own package and publish on npmjs instead of jsr.